tests/unit/customizations/cloudformation/test_deploy.py

```diff
@@ -365,6 +365,24 @@ def test_deploy_does_not_raise_exception_on_empty_changeset(self):
             fail_on_empty_changeset=False
         )
 
+    def test_deploy_empty_changeset_does_not_raise_exception_by_default(self):
```

Comment:
What's your thoughts on writing functional tests for these? I realize that there is no existing functional tests for this command set, but it would be really helpful to make sure everything is plumbed through correctly.